README.md

```diff
@@ -1,2 +1,6 @@
 # FlashFeetUI
 User Interface for team FlashFeet
+
+use npm start to run
```

Comment:
I feel this could be improved with the following: 

`
# Title Here...

1. Run `npm install` to build your application
2. Start application with `npm start`
3. To check that your application is running enter url `http://localhost:3000`
`